CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.3.1
+> Released on 2025/05/21
+
+### Dependencies
+* Sync platform-api to c82098d86d2345575281ec043c7d960e57fe63df
```

Comment:
What's the user facing impact of this sync?

Syncing to `platform-api` is our action. What does it mean for the user?